### PR TITLE
extract ListContent as a component

### PR DESCRIPTION
**Pull request by ZYSzys**
Extract `ListContent` as a component to reusable.

### CODE REVIEW

**Comment by chenshuai2144** [comment]
I don’t feel the meaning of the component, he is not universal

**Comment by ZYSzys** [comment]
I found the almost same `ListConent` appears on both [Account](https://github.com/ant-design/ant-design-pro/blob/536b28e182e7e59a66e48234a441e7bcf555c1f3/src/pages/Account/Center/Articles.js#L22) page and [List](https://github.com/ant-design/ant-design-pro/blob/536b28e182e7e59a66e48234a441e7bcf555c1f3/src/pages/List/Articles.js#L99) page, so maybe simply write once and invoke it twice when needed ?

**Comment by chenshuai2144** [comment]
How about changing the name?

like UserFooterContent

File: src/components/ListContent/index.less [hunk 1]
```diff
@@ -0,0 +1,38 @@
+@import '~antd/lib/style/themes/default.less';
+
+.listContent {
+  .description {
```
Comment (chenshuai2144) [inline src/components/ListContent/index.less]: `margin-top: 16px;` 
style get an error.
You need to add this.

**Comment by ZYSzys** [comment]
Yeah, we need to change the name indeed.
What about `ArticleListContent` since it’s just used in `Articles.js` ?

File: src/components/ListContent/index.less (hunk 1 above)
Comment (ZYSzys) [reply src/components/ListContent/index.less]: This file was cut from [Articles.less](https://github.com/ant-design/ant-design-pro/blob/536b28e182e7e59a66e48234a441e7bcf555c1f3/src/pages/List/Articles.less#L4) and I didn't change any style.
And `margin-top: 16px;` was added in the `.extra` under that.

It works well on my machine as below.
screen shot 2018-10-08 at 5 39 11 pm

Should I really need to change that ?

**Comment by chenshuai2144** [comment]
good job

**Pull request merged**